Lib/fractions.py

```diff
@@ -13,6 +13,12 @@
 __all__ = ['Fraction']
 
 
+def _gcd(a, b):
```

Comment:
Looks good, what about declared this function as the nested function?

Reply:
In this case, we can skip the news :)

Reply:
```diff
         self = super(Fraction, cls).__new__(cls)

+        def _gcd(a, b):
+            while b:
+                a, b = b, a%b
+            return a
+
         if denominator is None:
             if type(numerator) is int:
                 self._numerator = numerator
```

Reply:
> Looks good, what about declared this function as the nested function?

I'd prefer to revert to as close as possible to the previous code, which had `_gcd` as a module-level function.

I think we _do_ need a news entry, because this is fixing a bug that was already made available in a release. If the bug had been introduced and then fixed without any intermediate release, then we could have skipped the news entry.

Reply:
Got it :)